manager/api/views.py

```diff
@@ -1555,6 +1580,143 @@ def destroy(self, request, pk=None, *args, **kwargs):
         return Response(response.json(), status=response.status_code)
 
 
+@swagger_auto_schema(
+    method="post",
+    responses={
+        200: openapi.Response("Night report sent"),
+        400: openapi.Response("Night report already sent"),
+        401: openapi.Response("Unauthenticated"),
+        403: openapi.Response("Unauthorized"),
+    },
+)
+@api_view(["POST"])
+@permission_classes((IsAuthenticated,))
+def ole_send_night_report(request, *args, **kwargs):
+    """Confirm and mark a night report as sent
+
+    Params
+    ------
+    request: Request
+        The Request object
+
+    args: list
+        List of addittional arguments. Currently unused.
+
+    kwargs: dict
+        Dictionary with request arguments. Currently using the following keys:
+            pk (required): The primary key of the night report to be sent.
+
+    Returns
+    -------
+    Response
```

Comment:
Note that the numpy doc style does not require the return type to be in back quotes so this line is correct and should not be modified.